.gitignore

```diff
@@ -24,13 +24,11 @@ _testmain.go
 *.prof
 
 .DS_Store
-vendor/
 /test-tf
 terraform-provider-qingcloud*
-terraform-qingcloud*
-api.md
 /test-*
 *.tfstate
 build
 bin
-.idea/
\ No newline at end of file
+.idea/
+glide.lock
```

Comment:
Travis use gilde.lock to  download other dependencies , please modify travis.yml to ensure consistency of dependencies .

Reply:
since we've packed up all of dependencies and glide.lock is not used.